go.mod

```diff
@@ -14,43 +15,55 @@ require (
 	github.com/ipfs/go-datastore v0.6.0
 	github.com/ipfs/go-ds-badger3 v0.0.2
 	github.com/ipfs/go-log v1.0.5
-	github.com/libp2p/go-libp2p v0.24.1
-	github.com/libp2p/go-libp2p-kad-dht v0.20.0
-	github.com/libp2p/go-libp2p-pubsub v0.8.3
+	github.com/libp2p/go-libp2p v0.26.4
 	github.com/multiformats/go-multiaddr v0.8.0
 	github.com/prometheus/client_golang v1.14.0
 	github.com/rs/cors v1.8.3
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/viper v1.15.0
 	github.com/stretchr/testify v1.8.2
-	github.com/tendermint/tendermint v0.34.21
 	go.uber.org/multierr v1.10.0
 	golang.org/x/net v0.8.0
 	google.golang.org/grpc v1.52.0
 	google.golang.org/protobuf v1.30.0
 )
 
 require (
-	github.com/DataDog/zstd v1.4.5 // indirect
+	github.com/libp2p/go-libp2p-kad-dht v0.21.1
+	github.com/libp2p/go-libp2p-pubsub v0.9.3
+)
+
+require (
```

Comment:
Why? Maybe `go mod tidy` will fix this?

Reply:
sorry, what is this referring to, the require with the `// inderect`s, or the require above that?